data/brands/amenity/recycling.json

```
@@ -4,31 +4,12 @@
     "exclude": {
       "generic": [
         "^d[ée]ch[èe]tt?erie$",
-        "^recycling$"
+        "^recycling$",
+        "Bauhof", "Biotonne", "Glascontainer", "Recyclinghof", "Wertstoffhof"
```

These have to be regular expressions, so it would be formatted this to match a bunch of things:
```suggestion
        "^(bauhof|biotonne|glascontainer|recyclinghof|wertstoffhof)$"
```